async_firebase/client.py

```diff
@@ -108,6 +109,7 @@ def build_android_config(  # pylint: disable=too-many-locals
         title_loc_args: t.Optional[t.List[str]] = None,
         channel_id: t.Optional[str] = None,
         notification_count: t.Optional[int] = None,
+        visibility: t.Optional[Visibility] = None,
```

Comment:
Maybe this attribute could simply be optional, effectively this would be the same as setting it to an enumeration member "unspecified"